ExomeDepth/ed_csv_to_vcf.py

```diff
@@ -191,25 +193,26 @@ def cnv_locationtype(region, par1, par2):
             new_record.INFO['END'] = row['end']
             new_record.INFO['NTARGETS'] = row['nexons']
             new_record.INFO['SVLEN'] = int(row['end']) - int(row['start'])  # Input is assumed 0-based
-            new_record.INFO['CN'] = copynumber
+            new_record.INFO['REFLEN'] = int(row['end']) - int(row['start'])  # Input is assumed 0-based
             call_conrad = row['Conrad.hg19']
             if str(call_conrad) == "nan":
                 call_conrad = "NaN"
             new_record.INFO['cCNV'] = call_conrad
 
             """Change FORMAT fields"""
-            for f in ['GT', 'CCN', 'BF', 'RT', 'CR', 'RS', 'IH', 'CM', 'PD', 'TC']:
+            for f in ['GT', 'CN', 'BF', 'SM', 'CR', 'RS', 'IH', 'CM', 'PD', 'TC', 'BC']:
                 format_dict[f] = ""
             format_dict['GT'] = str(genotype)
-            format_dict['CCN'] = "%.2f" % (float(calc_copynumber))
+            format_dict['CN'] = "%.2f" % (float(calc_copynumber))
             format_dict['BF'] = "%.2f" % (float(row['BF']))
-            format_dict['RT'] = "%.2f" % (float(ratio))
+            format_dict['SM'] = "%.2f" % (float(ratio))
             format_dict['CR'] = "%.4f" % (float(row['correlation']))
```

Comment:
I think I would prefer `str(round(value, decimal))` but since calucation did not change it self you can leave it if you want...